draft-packages/button/KaizenDraft/Button/README.mdx

```diff
@@ -8,6 +8,14 @@ needToKnow:
 - If it needs to navigate somewhere and can be opened in a new tab, use a link instead.
 headerImage: button
 demoStoryId: button-zen-react--default-kaizen-site-demo
+health:
```

Comment:
:D Thanks, this looks good. 

👍🏻 American-English (internationalized)
👍🏻 internationalized category also represents RTL
👍🏻 designed and build categories are related to health. This holistically represents the component's health - including both code and design quality attributes.
👍🏻 "documented" omitted. This is redundant, as we don't typically create placeholder pages for upcoming components - if this page existed it's documented.

I could see a future where we _might_ want to add some nuance to these categories and even expand the categories themselves (has this component passed an accessibility audit, has this component undergone user testing, etc). I believe this format will support those needs nicely 👍🏻

Reply:
On the topic of passing an accessibility audit, I can see us wanting to include some dates in future:

```
lastDesignReviewed: "2021-07-10",
lastAccessibilityCertification: "2020-10-16",
```

or similar